tests/functional/strategy/test_config.py

```diff
@@ -64,13 +66,10 @@ def test_strategy_setParams(gov, strategist, strategy, rando, getter, setter, va
     with brownie.reverts():
         getattr(strategy, setter)(val, {"from": rando})
 
-    getattr(strategy, setter)(val, {"from": strategist})
-    assert getattr(strategy, getter)() == val
+    caller = {"gov": gov, "strategist": strategist}[caller]
 
-    if getter == "strategist":
-        # Strategist can't change themselves once they update
-        with brownie.reverts():
-            strategy.setStrategist(strategist, {"from": strategist})
```

Comment:
Still import to check this condition, but could do in another test case I suppose